app/src/main/java/dev/candar/swing/JFFile.java

```diff
@@ -28,9 +29,10 @@ class JFFile extends JPanel {
         gc.weightx = 1;
         gc.fill = GridBagConstraints.HORIZONTAL;
 
-        icon = new JLabel( 
-            new ImageIcon(Utils.iconToImage(isDirectory ? Resources.dirIcon : Resources.fileIcon)
-                .getScaledInstance(50, 50, Image.SCALE_FAST))
+        JLabel icon = new JLabel( 
+            (dim ?
```

Comment:
Nitpick: Multiple line ternary operators are not preferred. Using if statement here would be better for readability.